src/example/stokes_flow/stokes_flow.jl

```diff
@@ -0,0 +1,354 @@
+module stokes_flow_API #hide
```

Comment:
I suggest the following structure:
# Stokes flow example (FE with Taylor-Hood elements)
## Steady case : Moffat vortices
### Theory 
### ...
## Unsteady case : oscillating plate
### Theory 
### ...

We could discuss it.